### ISSUE TITLE
Should Create/Update activity return to Organisation page

### ISSUE THREAD

**Issue by wszarmach** [krico/jas#267]
Once the user has created or updated an activity they are not likely to continue changing that activity. Assuming no error occurs perhaps the user should be automatically redirected to the organisation screen.

**Comment by pauljanes** [comment]
Yes this was one thing I noticed as it isnt clear at present whether update was successful. I would imagine it to go back to the main screen if update successful
